app/feature/feature-help-center/src/main/kotlin/com/hedvig/android/feature/help/center/home/HelpCenterHomeDestination.kt

```diff
@@ -201,40 +187,83 @@ private fun HelpCenterHomeScreen(
                       verticalArrangement = Arrangement.Center,
                     ) {
                       Spacer(modifier = Modifier.height(12.dp))
-                      Text(
-                        text = when (quickLink) {
-                          is HelpCenterUiState.QuickLinkType.CommonClaimType -> quickLink.commonClaim.title
-                          is HelpCenterUiState.QuickLinkType.QuickActionType -> stringResource(
-                            quickLink.quickAction.titleRes,
-                          )
-                        },
-                        textAlign = TextAlign.Start,
-                        modifier = Modifier.padding(horizontal = 16.dp),
+                      Spacer(
+                        modifier = Modifier
+                          .height(24.dp)
+                          .padding(horizontal = 16.dp)
+                          .width(80.dp)
+                          .placeholder(true, highlight = PlaceholderHighlight.shimmer()),
                       )
                       Spacer(modifier = Modifier.height(4.dp))
-                      Text(
-                        text = when (quickLink) {
-                          is HelpCenterUiState.QuickLinkType.CommonClaimType -> {
-                            val hintTextRes = quickLink.commonClaim.hintTextRes
-                            if (hintTextRes != null) stringResource(hintTextRes) else ""
-                          }
-                          is HelpCenterUiState.QuickLinkType.QuickActionType -> stringResource(
-                            quickLink.quickAction.hintTextRes,
-                          )
-                        },
-                        textAlign = TextAlign.Start,
-                        modifier = Modifier.padding(horizontal = 16.dp),
-                        color = MaterialTheme.colorScheme.onSurfaceVariant,
-                        style = MaterialTheme.typography.titleSmall,
+                      Spacer(
+                        modifier = Modifier
+                          .height(20.dp)
+                          .padding(horizontal = 16.dp)
+                          .width(240.dp)
+                          .placeholder(true, highlight = PlaceholderHighlight.shimmer()),
                       )
                       Spacer(modifier = Modifier.height(12.dp))
                     }
                   }
                 }
               }
-            },
-          )
-        }
+              for (quickLink in quickLinks) {
```

Comment:
Since now we have this scenario where technically quickLinks and placeholders can show at the same time, since the state is possible to be in that combination, the change to a sealed interface will help with that too